src/components/User/Assets/AssetPairs/components/AssetPairManager.vue

```diff
@@ -153,6 +217,9 @@ export default {
     },
 
     async updatePolicy () {
```

Comment:
actually, not only policies being updated

Reply:
Renamed